### PR TITLE
45 rule fixes

### PR DESCRIPTION
**Pull request by yapei**

### CODE REVIEW

File: lib/rules/web/admin_console/4.5/common_ui_elements.xyaml [hunk 1]
```diff
@@ -427,12 +427,7 @@ uncheck_checkbox:
 check_link_and_text:
   element:
     selector:
-      xpath: //a[contains(.,'<text>')]
-  scripts: 
-    - command: |
-        var encodedhref = document.querySelector("a[data-test-id='<text>']").href
-        return decodeURIComponent(encodedhref).endsWith("<link_url>")
-      expect_result: true
+      xpath: //a[contains(.,'<text>') and contains(@href,'<link_url>')]
```
Comment (yapei) [inline lib/rules/web/admin_console/4.5/common_ui_elements.xyaml:430]: revert back the changes

File: lib/openshift/subscription.rb [hunk 2]
```diff
@@ -8,7 +8,7 @@ def current_csv(user: nil, quiet: false, cached: false)
     end
 
     def ready?(user:, quiet: false, cached: false)
-       res = get(user: user, cached: cached, quiet: quiet)
+       res = get(user: user, quiet: quiet)
```
Comment (yapei) [inline lib/openshift/subscription.rb:11]:
```
    def get(user: nil, quiet: false)
      user = default_user(user)
```
only two parameters are accepted for `get`

**Comment by yapei** [comment]
@liangxia  Can you help merge if it looks good to you?

File: lib/openshift/subscription.rb (hunk 2 above)
Comment (liangxia) [reply lib/openshift/subscription.rb:11]: There is one parameter unused in `ready`.

Comment (yapei) [reply lib/openshift/subscription.rb:11]: @liangxia  Updated and got passed jenkins log at #104672

**Pull request merged**